CONTRIBUTING.md

```diff
@@ -0,0 +1,315 @@
+    - [Questions](#questions)
+    - [Bug reports](#bug-reports)
+      - [Feature requests and contributions](#feature-requests-and-contributions)
+  - [Pull requests](#pull-requests)
+  - [Roadmap](#roadmap)
+  - [Developer environment setup](#developer-environment-setup)
+    - [Repository structure](#repository-structure)
+  - [Coding standards](#coding-standards)
+    - [General principles](#general-principles)
+    - [Formatting and linting](#formatting-and-linting)
+    - [Dependencies](#dependencies)
+    - [Tests](#tests)
+      - [Arrange, act, assert](#arrange-act-assert)
+    - [Type hints](#type-hints)
+    - [Documentation](#documentation)
+      - [Docstrings](#docstrings)
+      - [Building the documentation](#building-the-documentation)
+      - [Publishing the documentation](#publishing-the-documentation)
+    - [Changelog](#changelog)
+  - [Releasing](#releasing)
+
+<!-- include_in_docs_from_here -->
+
+## Introduction
+
+Thank you for considering contributing to `clinlp`! This project is open source and community driven, so it is largely dependent on contributors like you. This document should help you get involved with the project, whether reporting a bug or issue, requesting a feature, making changes to the codebase, or contributing in any other way.
+
+Please keep in mind that this page describes the ideal process and criteria for contributions. We understand that not all contributions will meet these criteria, and that is perfectly fine. We are happy to assist, so please don't hesitate to reach out to us.
+
+## Contact
+
+Our preferred way of communication is through [issues](https://github.com/umcu/clinlp/issues), GitHubs built-in issue tracker. We use it for most communication, including questions, bug reports, feature requests, help getting started, etc. This way, the entire community can benefit from the discussion. If this is not an option, you can also reach out to us by e-mail: [analytics@umcutrecht.nl](mailto:analytics@umcutrecht.nl).
+
+To create an issue right now, you can use the following link: [Create an issue](https://github.com/umcu/clinlp/issues/new).
+
+We will try to respond to you as soon as possible. Please keep in mind that we are with a small group of maintainers, so we might not always be able to get back to you within a few days.
+
+### Questions
+
+If you have any questions about the project, or need help getting started, please include at least the following information in your issue:
+
+- A clear and descriptive title
+- A description of the question
+- Any other relevant information
+
+### Bug reports
+
+If you encounter a bug, such as an error or unexpected behavior, please include at least the following information in your issue:
+
+- A clear and descriptive title
+- A detailed description of the bug
+- Steps to reproduce the bug (if possible, include a minimal code snippet)
+- Expected behavior
+- Actual behavior
+- If possible, include the version of `clinlp` you are using
```

Comment:
Dit is op zich heel handig om te weten, toch? Ik meen me nog te herinneren dat ikzelf een keer een oudere versie had gebruikt, waardoor ik een error krijg. Lijkt me ook makkelijk om even op te zoeken en mee te geven. Zou derhalve het 'If possible,' weglaten. Zelfde geldt voor de Python versie.